pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -109,6 +109,8 @@ func NewCmdNewBuild(fullName string, f *clientcmd.Factory, in io.Reader, out io.
 	cmd.Flags().StringVar(&config.ContextDir, "context-dir", "", "Context directory to be used for the build.")
 	cmd.Flags().BoolVar(&config.DryRun, "dry-run", false, "If true, do not actually create resources.")
 	cmd.Flags().BoolVar(&config.NoOutput, "no-output", false, "If true, the build output will not be pushed anywhere.")
+	cmd.Flags().StringVar(&config.SourceImage, "source-image", "", "Specify an image to use as source for the build.")
+	cmd.Flags().StringVar(&config.SourceImagePath, "source-image-path", "", "Specify the file or directory to copy from the source image and its destination in the build directory. Format: [source]:[destination-dir]")
```

Comment:
debatable whether this should have one since it could be confused as part of the example syntax, but i'll add it.